test/qtiCommonRenderer/interactions/textEntry/test.js

```diff
@@ -143,19 +139,15 @@ define([
 
                 $input.val('');
                 $input.focus();
-                assert.equal(getTooltipContent($input), __('This is not a valid answer'));
+                assert.ok(getTooltip($input).not(':visible'), 'the tooltip error is not hidden after a correct response');
```

Comment:
```suggestion
                assert.ok(getTooltip($input).not(':visible'), 'The error tooltip is shown on an empty input field');
```

Same should be applied above IMO.

BTW, what exactly does `getTooltip($input).not(':visible')` return? Are you sure it's a jQuery object?

Reply:
We are validating if the tooltip is visible or not, so it's better to put some text like `the error tooltip is hidden in a correct response` instead of your suggestion.

`getTooltip($input).not(':visible')` checks the visibility status with css and return `true` or `false`

Yes, It's JQuery

Reply:
OK, then `getTooltip($input).not(':visible')` will always return an object, and `assert.ok()` on it will always pass.
You should do `assert.ok(getTooltip($input).not(':visible').length)`.